lib/inspec/plugin/v2/loader.rb

```diff
@@ -59,6 +62,20 @@ def exit_on_load_error
       end
     end
 
+    def activate_mentioned_cli_plugins(cli_args = ARGV)
+      # Get a list of CLI plugin activation hooks
+      registry.find_activators(plugin_type: :cli_command).each do |act|
+        next if act.activated
+        # If there is anything in the CLI args with the same name, activate it
+        # If the word 'help' appears in the first position, load all CLI plugins
+        if cli_args.include?(act.activator_name.to_s) || cli_args[0] == 'help'
+          activate(:cli_command, act.activator_name)
+          act.implementation_class.register_with_thor
+        end
+        # rubocop: enable Lint/RescueException
```

Comment:
I see this enable here, but don't see the corresponding disable. Am I missing something?